packages/modules/user/src/services/user-module.ts

```diff
@@ -311,26 +315,16 @@ export default class UserModuleService
     }
 
     const toCreate = data.map((invite) => {
+      const id = generateEntityId((invite as { id?: string }).id, "invite")
```

Comment:
thought: Don't think we need this anymore, should be generated automatically no?

Reply:
I manually provide the id here to be able to generate the token without having to first create and then update the create tuples. We have the same approach in the fulfillment where we rely on the id before it gets created, that way it become known from there and allow us to cut down the back and forth